podcasts/ServerSyncManager.swift

```diff
@@ -73,6 +73,7 @@ class ServerSyncManager: ServerSyncDelegate {
         Analytics.shared.refreshRegistered()
         DispatchQueue.main.async {
             PlaybackManager.shared.effectsChangedExternally()
+            Theme.sharedTheme.toggleTheme()
```

Comment:
Theme is refreshed here. In testing, it didn't seem to matter if this was called each refresh vs. only when the theme changes.

Reply:
Would it make more sense to change `toggleTheme` to check this instead?

```swift
func toggleTheme() {
    let newTheme = toggledThemed()
    if activeTheme != newTheme {
        activeTheme = toggledThemed()
    }
}
```

Reply:
I guess so!